src/getHumanAge.test.js

```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+  test('return array = 0 if function is empty', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+  test('First 15 **cat** years give 1 **human** year', () => {
+    expect(getHumanAge(0, 15))
+      .toEqual([0, 1])
+  });
+  test('the next 9 **cat** years give 1 more **human** year', () => {
```

Comment:
Names of tests shouldn't be a continuation of a previous test. Every test should be independent, and the name of the test as well. You can use structure `... should be returned for the input ..` for example.